src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -18,11 +18,44 @@ import { UrlPathTemplate } from '../../../utils/urlPaths';
 import { authenticatedApiRequest } from '../../../api/apiRequest';
 import { getValueByKey } from '../../../utils/user-helpers';
 import { fetchPerson } from '../../../api/cristinApi';
+import { postOrcidCredentials } from '../../../api/orcidApi';
+import { OrcidCredentials } from '../../../types/orcid.types';
 
 interface UserOrcidProps {
   user: User;
 }
 
+function allPropertiesAreInCredentialsObject(rawCredentials: { [p: string]: string }) {
+  return (
```

Comment:
kan bruke inline arrow func her også, for consistency 👊 
```suggestion
const allPropertiesAreInCredentialsObject(rawCredentials: { [p: string]: string }) =>
  (
```

Reply:
Fikset